.pre-commit-config.yaml

```diff
@@ -49,7 +49,6 @@ repos:
     - --output-format=colorized
     - --score=n
     - --enable=local
-    - --disable=C0330,R0913,R0914
```

Comment:
Are these rules disabled in the new config? It's hard to know from numeric codes.

Reply:
I will check, thank you

Reply:
Marking as resolved.